tests/test_view.py

```diff
@@ -105,3 +108,27 @@ def test_view_share_updates_dictionary_not_overwrite():
     viewclass.share({'test2': 'test2'})
 
     assert viewclass.dictionary == {'test1': 'test1', 'test2': 'test2'}
+
+
+def test_view_cache():
+    """
+    Test for cache template each 5 min
+    """
+
+    container = App()
+
+    view = View().load_container(container)
+    view.share({'test': 'test'})
+
+    container.bind('CacheConfig', cache)
+    container.bind('CacheDiskDriver', CacheDiskDriver)
+    container.bind('CacheManager', CacheManager())
```

Comment:
The reason you are getting a Key Error for `Cache` is because you never loaded it into the container here. This section should look like:

```python
    container.bind('CacheConfig', cache)
    container.bind('CacheDiskDriver', CacheDiskDriver)
    container.bind('CacheManager', CacheManager(container))
    container.bind('Application', application)
    container.bind('Cache', container.make('CacheManager').driver('disk'))
    container.bind('View', view.render)
```

Notice the Application addition there since your `CacheDiskDriver` requires the `Application`. When you are building a container from scratch like that you'll need to add everything you need manually